common/src/main/java/io/pravega/common/util/TypedProperties.java

```diff
@@ -125,27 +125,35 @@ public boolean getBoolean(Property<Boolean> property) throws ConfigurationExcept
     }
 
     private <T> T tryGet(Property<T> property, Function<String, T> converter) {
-        String fullKeyName = this.keyPrefix + property.getName();
+        String propNewName = this.keyPrefix + property.getName();
+        String propOldName = this.keyPrefix + property.getLegacyName();
+        String propValue = null;
 
-        // Get value from config.
-        String value = this.properties.getProperty(fullKeyName, null);
+        if (property.hasLegacyName()) {
+            // Value of property with old name
+            propValue = this.properties.getProperty(propOldName, null);
```

Comment:
Let's add a `log.warn` to this code and notify users that they are using a deprecated property name. 

maybe something like 
`Deprecated property name '{}' used. Please use '{}' instead. Support for the old property name will be removed in a future version of Pravega.`

Reply:
Yes, that makes sense. I've added this comment now.